### PR TITLE
Define shorter timeout for ntp calls (2s) and tolerate failures

### PR DESCRIPTION
**Pull request by dshulyak**
I noticed that ntp fails quite often, cause usually there is no sla guarantees for servers that are used in ntp pool.

### CODE REVIEW

File: timesource/timesource.go [hunk 1]
```diff
@@ -16,13 +16,19 @@ const (
 	DefaultServer = "pool.ntp.org"
 
 	// DefaultAttempts defines how many servers we will query
-	DefaultAttempts = 3
+	DefaultAttempts = 5
+
+	// DefaultTolerateErrors defines how many failures will be tolerated.
+	DefaultTolerateErrors = 2
```
Comment (mandrigin) [inline timesource/timesource.go]: I would call it ‘MaxAllowedFailures’ or something like that

**Review by mandrigin — APPROVED** [review]
Looks good!

**Review by adambabik — APPROVED** [review]

**Comment by mandrigin** [comment]
@dshulyak do you plan to tweak this PR more or can I merge it?

**Comment by dshulyak** [comment]
> @dshulyak do you plan to tweak this PR more or can I merge it?

i will change variable name. but you can't merge it because of the failed CI :)
i think thats because of the cache

**Comment by mandrigin** [comment]
@dshulyak it looks different this time... the node wasn't able to download anything at all. See "highest=0" everywhere. And a failure in the end.

```
513.88s$ ./build/bin/statusd -datadir=.ethereumtest/Rinkeby -les -networkid=4 -sync-and-exit=30 -log=WARN -standalone=false -discovery=false
WARN [05-07|09:50:21] Light client mode is an experimental feature 
WARN [05-07|09:51:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:52:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:53:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:54:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:55:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:56:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:57:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0
WARN [05-07|09:58:21] Synchronization is not finished          package=status-go/geth/node.StatusNode current=7360 highest=0

ERROR[05-07|09:58:54] Light Ethereum peer registration failed  id=28ecf5272b560ca9 conn=staticdial err="peer set is closed"
```

**Pull request merged**